### PR TITLE
chore: address final autorefresh feedback

### LINKED ISSUE #1570: Address Final AutoRefresh Feedback
After meeting with @daviddkkim last week, we achieved a consensus about the final design implementation for the initial version of the dashboard auto refresh feature. This ticket represents the tracking of the work needed to get that design across. 

The points discussed:
 * Make the button on the right
 * Make the typeable inside the overlay
 * The overlay controls the behavior
 * More options for inactivity timeout
 * Button is purple while timeout is active

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #1570

This incorporates final feedback from @daviddkkim regarding the intended appearance and structure of the autorefresh feature
Screen Shot 2021-06-01 at 11 25 24 AM
Screen Shot 2021-06-01 at 11 25 32 AM
Screen Shot 2021-06-01 at 11 25 41 AM

### CODE REVIEW

File: src/dashboards/components/AutoRefreshInput.tsx [hunk 1]
```diff
@@ -64,15 +55,17 @@ const AutoRefreshInput: FC = () => {
...
-      dispatch(setAutoRefreshInputValue(currentDashboardId, selection))
+
+      setRefreshContext({
+        type: 'SET_REFRESH_MILLISECONDS',
+        refreshMilliseconds: {
+          interval: calculatedMilliseconds,
+          status: AutoRefreshStatus.Active,
+          label: selection,
+        },
+      })
```
Comment (Asalem1) [inline src/dashboards/components/AutoRefreshInput.tsx:66]: so happy these are coupled and we're reducing this to a single dispatch

**Review by Asalem1 — APPROVED** [review]
7th heaven

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1569)

**Review by daviddkkim — APPROVED** [review]
lgtm. There is an issue in the quick hits to address visual discrepancies but that shouldn't hold this back. This is good to go forward

**Pull request merged**